test/experimental/test_with_asset.py

```diff
@@ -178,23 +178,22 @@ def test_glove_different_dims(self):
     def test_vocab_from_file(self):
         asset_name = 'vocab_test.txt'
         asset_path = get_asset_path(asset_name)
-        v = load_vocab_from_file(asset_path, unk_token='<new_unk>')
-        expected_itos = ['<new_unk>', 'b', 'a', 'c']
+        v = load_vocab_from_file(asset_path)
+        expected_itos = ['b', 'a', 'c']
         expected_stoi = {x: index for index, x in enumerate(expected_itos)}
         self.assertEqual(v.get_itos(), expected_itos)
         self.assertEqual(dict(v.get_stoi()), expected_stoi)
 
     def test_vocab_from_raw_text_file(self):
         asset_name = 'vocab_raw_text_test.txt'
         asset_path = get_asset_path(asset_name)
-        tokenizer = basic_english_normalize()
-        jit_tokenizer = torch.jit.script(tokenizer)
-        v = build_vocab_from_text_file(asset_path, jit_tokenizer, unk_token='<new_unk>')
-        expected_itos = ['<new_unk>', "'", 'after', 'talks', '.', 'are', 'at', 'disappointed',
-                         'fears', 'federal', 'firm', 'for', 'mogul', 'n', 'newall', 'parent',
-                         'pension', 'representing', 'say', 'stricken', 't', 'they', 'turner',
-                         'unions', 'with', 'workers']
+        v = build_vocab_from_text_file(asset_path)
+        expected_itos = ['after', 'talks', "'disappointed'", 'Fears', 'Federal',
```

Comment:
What happened to the  "'" in the previous expected_itos?

Reply:
Actually we changed the tokenizer to python split() function to get away with the dependency on jited tokenizer as they are only available in experimental. This changed the order and expected words for the new vocab constructed using split() function tokenizer.

Reply:
The basic_english_normalize() tokenizer/normalizer broke the 'disappointed' into "'", "disappointed" and "'" which is not the case with split()

Reply:
Hm, but using _build_vocab_from_text_file_using_python_tokenizer below this should work now even if it's not JIT'd right?

Reply:
Yes, now we support both JIT'd and pure Python object as tokenizers. Added corresponding tests to check it works with both the cases.